Automattic/jetpack
# Jetpack Social: Add posts box

Adds a posts box to the Jetpack Social admin page, showing many posts have been Publicized in the past 30 days.

#### Changes proposed in this Pull Request:

* Add posts box to Jetpack Social admin page.

#### Other information:


#### Jetpack product discussion


1202436584533778-as-1202583289493841
/f
#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Make sure the share limit isn't enabled for your site.
* Go to the Jetpack Social admin page.
* Check if the boxes display properly.
* If both numbers are 0, Publicize a post to two or more connections, and make sure the number of posts in the posts box only increases by 1.

<img width="1146" alt="image" src="https://user-images.githubusercontent.com/1713699/180430825-da446807-5dfa-434e-a4cb-641bbc5c5126.png">
